src/components/Contacts/ContactDetails/ContactTaskRow/TaskCommentsButton/TaskCommentsButton.tsx

```diff
@@ -0,0 +1,48 @@
+import { Button, styled, Theme, Typography } from '@material-ui/core';
+import { ChatBubbleOutline } from '@material-ui/icons';
+import React from 'react';
+
+const TaskRowWrap = styled(Button)(({ theme }) => ({
+  display: 'flex',
+  justifyContent: 'center',
+  alignItems: 'center',
+  height: 32,
+  width: 58,
+  borderRadius: 4,
+  border: `1px solid ${theme.palette.text.secondary}`,
+  margin: theme.spacing(2),
+}));
+
+const TaskCommentIcon = styled(ChatBubbleOutline)(({ theme }) => ({
+  color: theme.palette.text.secondary,
+}));
+
+const TaskCommentNumber = styled(Typography)(
+  ({ theme, isComplete }: { theme: Theme; isComplete: boolean }) => ({
+    color: isComplete
+      ? theme.palette.text.secondary
+      : theme.palette.text.primary,
+    margin: theme.spacing(0.5),
+  }),
+);
+
+interface TaskCommentsButtonProps {
+  isComplete: boolean;
+  numberOfComments: number;
+  onClick: () => void;
+}
+
+export const TaskCommentsButton: React.FC<TaskCommentsButtonProps> = ({
+  isComplete,
+  numberOfComments = 0,
+  onClick,
+}) => {
+  return (
+    <TaskRowWrap onClick={() => onClick()}>
```

Comment:
I was doing this because the `onClick` property expects a function with `Event` as a parameter...  and I wanted to avoid adding that to the callback function because it doesn't use it.  But I agree that this is not as clean to look at.